docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -0,0 +1,173 @@
+
+|Option|Type|Description|
+|---|---|---|
+|`name`|String|The name of the require policy|
+|`filter`|[Filter](./moderated-sessions.mdx#filters)|An expression that, if it evaluates to true for a given user, enables the user to be present in a Moderated Session|
+|`kinds`|`[]`[Session kind](./moderated-sessions.mdx/#session-kinds)|The kind of session that the policy applies to|
+|`modes`|`[]`[Participant mode](./moderated-sessions.mdx#participant-modes)|The participant mode that applies to the user joining the Moderated Session under this policy|
+|`count`|[Participant mode](./moderated-sessions.mdx#require-policy-count)|The amount of users that need to match the filter expression to satisfy the policy|
+
+#### Example
+
+The policy below specifies that a given role
+must have a minimum of two users with the role `auditor` and the mode `moderator` present in the session
+to start it. The policy applies to SSH and Kubernetes sessions only.
+
+When a user with this require policy starts a session, it will be pending
+until the policy is fulfilled. 
+
+```yaml
+kind: role
+metadata:
+  name: prod-access
+spec:
+  allow:
+    require_session_join:
+      - name: Auditor oversight
+        filter: 'contains(user.roles, "auditor")'
+        kinds: ['k8s', 'ssh']
+        modes: ['moderator']
+        count: 2
+```
+
+### `join_sessions`
+
+#### Options
+
+The following are required options for `join_sessions`:
+
+|Option|Type|Description|
+|---|---|---|
+|`name`|String|The name of the allow policy|
+|`roles`|[]String|A list of names for Teleport roles that this policy applies to. Users with this role are eligible to join a Moderated Session under this policy.|
+|`kinds`|`[]`[Session kind](./moderated-sessions.mdx#session-kinds)|The kind of session that the policy applies to|
+|`modes`|`[]`[Participant mode](./moderated-sessions.mdx#participant-modes)|The participant mode that paplies to the user joining the Moderated Session under this policy|
+
+#### Example
+
+The following allow policy attaches to the role `auditor` and allows one to join
+SSH and Kubernetes sessions started by a user with the role `prod-access` as a moderator or observer.
+
+```yaml
+kind: role
+metadata:
+  name: auditor
+spec:
+  allow:
+    join_sessions:
+      - name: Auditor oversight
+        roles : ['prod-access']
+        kinds: ['k8s', 'ssh']
+        modes: ['moderator', 'observer']
+```
+
+### Filters
+
+A filter expression or predicate is a string statement used to define logic based on a set of input variables.
+
+Filter expressions  allow for more detailed control.
+`user.name == "adam" || contains(user.roles, "cs-observe")` is a simple example of a filter expression
+that evaluates to true if the user is Adam or if the user has the trait `cs-observe`.
+
+Require policies can specify which users they consider as valid with a filter expression. They work like
+other filters used within Teleport such with access requests. The filter context has a `user` object defined
```

Comment:
```suggestion
 The filter context has a `user` object defined
```

Suggested removing this sentence since we don't have good docs elsewhere on filter expressions.